src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/hmctsresponsereviewed/HmctsResponseReviewedAboutToSubmitHandlerTest.java

```diff
@@ -48,13 +46,14 @@ public class HmctsResponseReviewedAboutToSubmitHandlerTest {
     @Before
     public void setUp() {
         openMocks(this);
-        dwpDocumentService = new DwpDocumentService();
+        DwpDocumentService dwpDocumentService = new DwpDocumentService();
         handler = new HmctsResponseReviewedAboutToSubmitHandler(dwpDocumentService);
 
         when(callback.getEvent()).thenReturn(EventType.HMCTS_RESPONSE_REVIEWED);
         when(callback.getCaseDetails()).thenReturn(caseDetails);
         sscsCaseData = SscsCaseData.builder().ccdCaseId("ccdId").appeal(Appeal.builder().build())
-                .selectWhoReviewsCase(new DynamicList(new DynamicListItem("reviewByTcw", "Review by TCW"), null))
+                .selectWhoReviewsCase(new DynamicList(
+                    new DynamicListItem("reviewByTcw", "Review by TCW"), null))
```

Comment:
in the future avoid making changes where not relevant as you've made alot of formatting changes to this file it bloats the PR

Reply:
Personally I think its good to cleanup files which you work in, it cleans up the codebase fairly well and means you always leave it in a nicer state than it was. Given that some of these files I didn't make any functional changes and I will be mindful of changing them in the future